.github/workflows/tests.yml

```
@@ -22,3 +22,11 @@ jobs:
         run: make -C components/tls test
       - name: test-components-kafka
         run: make -C components/kafka test
+      - name: check CRDs changes
+        run: | 
+          make -C operator generate
```

```suggestion
          make -C operator generate manifests
```
the manifests target is also required